sunpy/net/helioviewer.py

```diff
@@ -101,20 +92,9 @@ def download_jp2(self, date, directory=None, overwrite=False, **kwargs):
         ----------
         date : `datetime.datetime`, string
             A string or datetime object for the desired date of the image
-        directory : string
-            (Optional) Directory to download JPEG 2000 image to.
```

Comment:
Do we still not support this?

Reply:
As we discussed earlier we don't need to explain optional parameters if we are not letting the user to add optional parameters when they call the fucntion as we are not using **kwargs so I removed them